tape/src/main/java/com/squareup/tape/FileObjectQueue.java

```diff
@@ -39,75 +33,59 @@ public FileObjectQueue(File file, Converter<T> converter) throws IOException {
     this.queueFile = new QueueFile(file);
   }
 
+  public File file() {
```

Comment:
`FileException` exposed the underlying `File`. This allows calling code to retrieve the `File` for exception reporting purposes.